lib/screens/create_region_screens/select_region/select_region.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:seeds/components/flat_button_long.dart';
+import 'package:seeds/utils/build_context_extension.dart';
+
+class SelectRegion extends StatelessWidget {
```

Comment:
is this part of this pr?

Reply:
No, the other code was not yet merge. I need to merge this prs in order